Linux compile failing

<!-- place 'x' mark between square [] brackets to checkmark box -->

**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated


<!-- Issues will be closed without being looked into if the following information is missing (unless its not applicable). -->

**_Client Version_** (type `/ver` in game) **:** 
N/A

**_Source Branch_** (master/stable) **:** 
master

<!-- If there is a server you know we can reproduce this on right now, please mention it here. -->
**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
git clone https://github.com/DarkStarProject/darkstar.git
cd darkstar
sh autogen.sh &> autogenlog.log
./configure --enable-debug=gdb > configurelog.log
make -j 12 &> makelog.log

compile fails with the following error:
/usr/bin/ld: dsgame-instance_loader.o: undefined reference to symbol 'pthread_create@@GLIBC_2.2.5'
/usr/bin/ld: /usr/lib/libpthread.so.0: error adding symbols: DSO missing from command line

My system
gcc: 9.2.1
cmake: 3.16.4-1
zeromq: 4.3.2-1
mariadb: 10.4.12-1
luajit: 2.0.5-2
automake: 1.16.1-2
autoconf: 2.69-7
pkgconf: 1.6.3-3

distribution: Manjaro Linux 19.0.1-1
kernel: linux-tkg-pds-zen 5.5.6-17

[autogenlog.log](https://github.com/DarkstarProject/darkstar/files/4284457/autogenlog.log)
[configurelog.log](https://github.com/DarkstarProject/darkstar/files/4284459/configurelog.log)
[makelog.log](https://github.com/DarkstarProject/darkstar/files/4284460/makelog.log)

--- a/Makefile.am
+++ b/Makefile.am
@@ -22,7 +22,7 @@ LIBS_ALL                += -lzmq
 
 ## Add Architecture-specific stuff
 if DARKSTAR_ARCH_LINUX
-LIBS_ALL                += -ldl
+LIBS_ALL                += -ldl -lpthread
 endif
 
 if DARKSTAR_ARCH_SOLARIS

